.gitignore

```diff
@@ -8,4 +8,5 @@
 **/.client_secret
 **/.pytest_cache
 .mypy_cache
-.ruff_cache
\ No newline at end of file
+.ruff_cache
+**/.nvm/
```

Comment:
not sure we should be modifying the `.gitignore` for a change like this, why was this line needed?

Reply:
Good catch on the manifest, updated. Reverted the .gitignore updates.